packages/core/typings/index.d.ts

```diff
@@ -1,11 +1,8 @@
-import './src/prettier-plugin-solidity';
-import './src/remix-debug-debugtest';
+export { Logger } from 'embark-logger';
```

Comment:
I'm not sure I agree with this here...

Generating typings for every package is fine, but they should be exported from their dedicated packages. So instead of exporting `Logger` here, it should always be imported from `embark-logger`

Reply:
I agree, but the problem is that we have some circularity, and this is a way to deal with it until we figure out how to rearrange some things in `embark-core`, `embark-utils`, etc. such that we can move the definitions in `packages/core/typings` (package name `@types/embark`) into `embark-core`. Even then 

On `master` presently we have these dep relationships:
```
embark-core > embark-logger, embark-utils
embark-logger > embark-utils
embark-utils > embark-logger
```

On this branch:
```
embark-utils
embark-logger > embark-utils
embark-core > embark-logger, embark-utils
```

But `embark-utils` still wants the `Logger` type. So I introduced:
```
@types/embark > embark-logger
```

And then `embark-utils` in its `longRunningProcessTimer.ts ` can get  `{ Logger } from 'embark'` per the tsc magic of `@types` packages and the re-export here in `packages/core/typings/index.d.ts `.

I noticed the circularity problem when re-enabling typechecking in `embark-core` on this branch.

My solution is not elegant, but I feel like moving away completely from the `@types/embark` setup should happen in a future refactor PR.

I'm definitely open to suggestions on another approach in this PR!

Reply:
@PascalPrecht and I discussed this matter on a call earlier today and agreed that my solution is okay in the interim, i.e. until we can work through another typings refactor. This same interim solution will land in a PR intended to replace this one, which will be closed after I post this comment.